wiki/Beatmap_Editor_Guides/osu!mania_Mapping_Guide/en.md

```diff
@@ -47,7 +47,7 @@ Alright! Let’s drag the .mp3 of the song we want to map into osu! and a new ma
 
 #### General
 
-![](Song Setup.png)
+![](Song_Setup.png)
```

Comment:
L50: Put a text inside ``[]`` to inidcate which should be there if the image fails to load